x-pack/legacy/plugins/monitoring/server/lib/logstash/get_paginated_pipelines.js

```diff
@@ -31,11 +31,12 @@ export async function getPaginatedPipelines(
   req,
   lsIndexPattern,
   { clusterUuid, logstashUuid },
-  metricSet,
+  { latestThroughput: throughputMetric, latestNodesCount: nodesCountMetric },
```

Comment:
This seems like a strange change. I suggest we make this generic (leave `metricSet`) or just remove it as a parameter entirely

Reply:
Good call 👍